chain/client/src/test_utils.rs

```diff
@@ -565,7 +563,7 @@ pub fn setup_mock_all_validators(
     >,
 ) -> (Block, Vec<(Addr<ClientActor>, Addr<ViewClientActor>)>, Arc<RwLock<BlockStats>>) {
     let peer_manager_mock = Arc::new(RwLock::new(peer_manager_mock));
-    let validators_clone = validators.clone();
+    let validators = vs.all_block_producers().cloned().collect::<Vec<_>>();
```

Comment:
I think here validators should include chunk only producers too (instead of just block producers). If it doesn't include chunk only producers, then the code later won't construct ClientActor and ViewClientActor for chunk only producers, so the network may not have chunk only producers at all.

Reply:
Uhu. I originally planned to postpone this to PR which actually adds a test here, but, yeah, it makes more sense to fix it righth away, fixed in the last commit: 

https://github.com/near/nearcore/pull/7280/commits/47c1855fb210e691a6b2b051358e7e9c851c714b